src/ROUTES.js

```diff
@@ -78,17 +78,29 @@ export default {
     LOGIN_WITH_VALIDATE_CODE_2FA_WORKSPACE_CARD: 'v/:accountID/:validateCode/2fa/workspace/:policyID/card',
     ENABLE_PAYMENTS: 'enable-payments',
     WORKSPACE: 'workspace',
-    WORKSPACE_CARD: ':policyID/card',
```

Comment:
I think we can also remove the `WORKSPACE` route here, it doesn't seem like it's used anywhere else.

Reply:
Yep, good catch.